ngraph/frontend/onnx_import/src/op/gather_nd.cpp

```diff
@@ -30,11 +30,12 @@ namespace ngraph
             {
                 OutputVector gather_nd(const Node& node)
                 {
-                    OutputVector ng_inputs{node.get_ng_inputs()};
-                    auto data = ng_inputs.at(0);
-                    auto indices = ng_inputs.at(1);
+                    const OutputVector ng_inputs{node.get_ng_inputs()};
+                    const auto data = ng_inputs.at(0);
+                    const auto indices = ng_inputs.at(1);
+                    const int batch_dims = node.get_attribute_value<int>("batch_dims", 0);
```

Comment:
Please check if it will work for you.
```suggestion
                    const int batch_dims = node.get_attribute_value<int64_t>("batch_dims", 0);
```